content/metadata.yaml

```diff
@@ -70,6 +70,39 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2017-05-26
+  - github: jlevy44
+    name: Joshua J. Levy
+    orcid: 0000-0001-8050-1291
+    email: joshua.j.levy.gr@dartmouth.edu
+    affiliations:
+      - Program in Quantitative Biomedical Sciences, Geisel School of Medicine at Dartmouth, Lebanon, NH
+    funders: Burroughs Wellcome Fund Big Data in the Life Sciences training grant at Dartmouth
+    v1: "Drafted one or more subsections"
```

Comment:
Can you switch these from v1 which should be an empty string to v2?

Reply:
Okay I tried to give it a shot. I wasn't sure if the date was supposed to be when our PRs were merged.

Reply:
Sure, just made the change.